v2-migration/src/main/resources/META-INF/rewrite/change-s3-methods.yml

```diff
@@ -196,12 +196,6 @@ recipeList:
   - org.openrewrite.java.ChangeMethodName:
       methodPattern: com.amazonaws.services.s3.model.BucketCrossOriginConfiguration withRules(com.amazonaws.services.s3.model.CORSRule...)
       newMethodName: withCorsRules
-  - org.openrewrite.java.ChangeMethodName:
-      methodPattern: com.amazonaws.services.s3.model.ListObjectsRequest withBucketName(String)
-      newMethodName: withBucket
-  - org.openrewrite.java.ChangeMethodName:
-      methodPattern: com.amazonaws.services.s3.model.ListObjectsV2Request withBucketName(String)
-      newMethodName: withBucket
```

Comment:
Removing these as they are redundant with recent recipe addition to remove in V1SetterToV2

https://github.com/aws/aws-sdk-java-v2/blob/master/v2-migration/src/main/java/software/amazon/awssdk/v2migration/V1SetterToV2.java#L112